wire-library/wire-tests/src/commonTest/shared/json/all_64_max_proto3.json

```diff
@@ -44,5 +44,20 @@
     "9223372036854775807",
     "9223372036854775807"
   ],
-  "oneofInt64": "9223372036854775807"
+  "oneofInt64": "9223372036854775807",
+  "mapInt64Int64": {
+    "9223372036854775807": "9223372036854775807"
+  },
+  "mapInt64Uint64": {
+    "9223372036854775807": "9223372036854775807"
```

Comment:
this is the max signed long . . . use the max unsigned long to exercise the difference?